app/views/dashboard/_flag_list.html.erb

```diff
@@ -0,0 +1,38 @@
+<div>
+	<% if logged_in_as(['admin','moderator']) %>
```

Comment:
Hi, would you mind making all the indents regular? Just two spaces, please!

Reply:
Yes sir most definitely!

Reply:
Okay so what do we do about the travis thing? Did it work out?